### PR TITLE
Changed logging to stdout in fetch_peps_nacp_id

### PR DESCRIPTION
**Pull request by EmmaSamoilova**

### CODE REVIEW

File: business_register/management/commands/fetch_peps_nacp_id.py [hunk 1]
```diff
@@ -45,15 +42,15 @@ def is_same_full_name(relative_data, pep, declaration_id):
...
                              f'with NACP id {declaration_id}')
                 return False
             last_name = splitted_names[0]
             first_name = splitted_names[1]
             if len(splitted_names) == 3:
                 middle_name = splitted_names[2]
     if not last_name or not first_name:
-        logger.error(f'Check related person data ({relative_data}) from declaration '
+        stdout.write(f'Check related person data ({relative_data}) from declaration '
                      f'with NACP id {declaration_id}')
```
Comment (roman-tiukh) [inline business_register/management/commands/fetch_peps_nacp_id.py:54]: self.stdout.write

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/management/commands/fetch_peps_nacp_id.py [hunk 2]
```diff
@@ -28,44 +23,6 @@
 #  'postCode', 'passport_extendedstatus']
```
Comment (IegorSoboliev) [inline business_register/management/commands/fetch_peps_nacp_id.py:25]: better leave that here. We can use this function in another module

**Review by IegorSoboliev — CHANGES_REQUESTED** [review]

File: business_register/management/commands/fetch_peps_nacp_id.py (hunk 2 above)
Comment (roman-tiukh) [reply business_register/management/commands/fetch_peps_nacp_id.py:25]: in this case, you need to remove logging from the function and process them outside the function

File: business_register/converter/declaration.py [hunk 3]
```diff
@@ -732,14 +732,15 @@ def save_spouse(self, relatives_data, pep, declaration):
...
                                 relative_data,
-                                spouse_from_our_db,
-                                declaration.id
-                        ):
-                            break
-                        else:
+                                spouse_from_our_db
+                            )
                             spouse = spouse_from_our_db
+                        except ValueError:
```
Comment (IegorSoboliev) [inline business_register/converter/declaration.py:741]: What actually can lead to a ValueError?

Comment (EmmaSamoilova) [reply business_register/converter/declaration.py:741]: DO you mean KeyError is better?

Comment (roman-tiukh) [reply business_register/converter/declaration.py:741]: own error is better

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

**Review by IegorSoboliev — APPROVED** [review]

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**